<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: white;
}

QPushButton { 
	background-color: rgba(174, 241, 255,200);
}

QTreeWidget{
	outline:0px;
	background: #e8ffe8;
	color:#07689f;
	selection-background-color: transparent; 
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>20</width>
      <height>580</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>840</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>130</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>开门</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>电子模块智能租借系统</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeButton">
       <property name="text">
        <string>更改</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>821</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="homeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>801</width>
       <height>451</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="homePage">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>主页</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ps1_2Image">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="ps1_2label">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>110</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ps2_2Image">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>140</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="ps3_2Image">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>270</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="ps2_2label">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>240</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
      <widget class="QLabel" name="ps3_2label">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>370</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ps3_1Image">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>270</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="ps3_1label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>370</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
      <widget class="QLabel" name="ps2_1label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>240</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ps2_1Image">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="ps1_1label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>110</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ps1_1Image">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rentPage">
      <widget class="QTreeWidget" name="leaseTreeWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>801</width>
         <height>461</height>
        </rect>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <column>
        <property name="text">
         <string>模块名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>模块的数量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>时间</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="historyPage">
      <widget class="QTreeWidget" name="historyTreeWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>801</width>
         <height>461</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>租借的用户</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>模块名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>模块数量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>时间</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="debugLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>90</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>460</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>110</y>
      <width>147</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: rgb(174, 241, 255);
	border-radius: 20px;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="homepageButton">
          <property name="text">
           <string>主页</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: rgb(174, 241, 255);
	border-radius: 20px;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="leaseButton">
          <property name="text">
           <string>个人租借查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: rgb(174, 241, 255);
	border-radius: 20px;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="historyButton">
          <property name="text">
           <string>近期历史查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="usingLabel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>440</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>usering</string>
    </property>
   </widget>
   <widget class="QLabel" name="pageImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-image: url(:/image/gameImage.png);
    background-position: center;
    background-repeat: no-repeat;
    }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
